### PR TITLE
A new command for Web MQTT connection listing #10693 #9302

### PR DESCRIPTION
**Pull request by michaelklishin**
This is #10693 by @LoisSotoLopez with the necessary Bazel build system updates.

Closes #9302.

### CODE REVIEW

**Comment by michaelklishin** [comment]
The new command suite in mixed cluster version still fails with

```
=== === Reason: no match of right hand side value 
                 [{badrpc,
                      {'EXIT',
                          {undef,
                              [{rabbit_web_mqtt_app,
                                   emit_connection_info_local,
                                   [[],#Ref<0.4241247191.103809025.44488>,
                                    <0.146.0>],
                                   []}]}}}]
  in function  command_SUITE:run/1 (command_SUITE.erl, line 84)
```

which is a build system-level issue.

We will either fix it or remove the mixed cluster version. This command is new and we can expect that it won't be executed before all cluster nodes are upgraded (at least this is not a risky assumption to make).

File: deps/rabbitmq_web_mqtt/test/command_SUITE.erl [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+end_per_suite(Config) ->
+    rabbit_ct_helpers:run_teardown_steps(Config,
+      rabbit_ct_client_helpers:teardown_steps() ++
+      rabbit_ct_broker_helpers:teardown_steps()).
+
+init_per_group(unit, Config) ->
+    Config;
+init_per_group(Group, Config) ->
+    case rabbit_ct_helpers:is_mixed_versions() of
```
Comment (dumbbell) [inline deps/rabbitmq_web_mqtt/test/command_SUITE.erl:57]: Could you please check a real condition that the old version can't be used as part of this test?

`is_mixed_versions()` will still return true in 10 years when testing RabbitMQ 21.x against 22.x. This function should almost never be used.

**Review by dumbbell — CHANGES_REQUESTED** [review]

File: deps/rabbitmq_mqtt/test/command_SUITE.erl [hunk 2]
```diff
@@ -57,7 +57,23 @@ init_per_group(unit, Config) ->
...
+                lists:member(rabbit_web_mqtt_app, NodeApps)
+            end, AllAppNames) of
+                true ->
+                    Config1;
+                false ->
+                    {skip, "rabbit_web_mqtt_app not available on all nodes"}
+            end;
+        v5 ->
+            util:maybe_skip_v5(Config1)
+    end.
```
[the remark this answers is not in the document]
Comment (ansd) [reply deps/rabbitmq_mqtt/test/command_SUITE.erl:76]: fixed in https://github.com/rabbitmq/rabbitmq-server/pull/10825
Whether the Web MQTT plugin in enabled or disabled is completely unrelated to whether this test should be skipped.

File: deps/rabbitmq_mqtt/include/rabbit_mqtt.hrl [hunk 3]
```diff
@@ -12,7 +12,10 @@
 -define(PERSISTENT_TERM_EXCHANGE, mqtt_exchange).
 -define(DEFAULT_MQTT_EXCHANGE, <<"amq.topic">>).
 -define(MQTT_GUIDE_URL, <<"https://rabbitmq.com/docs/mqtt/">>).
+-define(WEB_MQTT_GUIDE_URL, <<"https://rabbitmq.com/docs/web-mqtt/">>).
```
[the remark this answers is not in the document]
Comment (ansd) [reply deps/rabbitmq_mqtt/include/rabbit_mqtt.hrl:15]: Fixed in https://github.com/rabbitmq/rabbitmq-server/pull/10825

File: deps/rabbitmq_web_mqtt/test/command_SUITE.erl (hunk 1 above)
Comment (ansd) [reply deps/rabbitmq_web_mqtt/test/command_SUITE.erl:57]: Fixed in https://github.com/rabbitmq/rabbitmq-server/pull/10825

**Pull request merged**